test/metabase/api/notify_test.clj

```diff
@@ -49,19 +49,18 @@
                     (catch clojure.lang.ExceptionInfo e
                       (select-keys (ex-data e) [:status :body]))))))
       (testing "table name must exist or we get a 404"
```

Comment:
Removing the body as it is now more explicit if the table is missing. It might make the tests flaky to add the specific formatted string in here since we really only care that it's a 404.